app/controllers/users.js

```diff
@@ -68,3 +70,15 @@ exports.signin = (req, res, next) => {
     next(err);
   });
 };
+
+exports.signout = (req, res, next) => {
+  bcrypt.genSalt(saltRounds).then((authenticationCode) => {
```

Comment:
To maintain the idea of the previous PR I would move the logic to a service instead of having it in the controller.